parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -150,12 +150,25 @@ EOF
 }
 
 {{- if NeedsContainerd}}
+
+# this check is outside because installStandaloneContainerd is also called by install_dependencies.sh in VHD Builder
+{{if HasContainerdVersion}}
+    echo "Containerd version specified by RP"
+{{else}}
+    echo "Containerd version not specified, use hardcoded version"
+{{end}}
```

Comment:
no, this doesn't make sense. The logging should be part of  installStandaloneContainerd

if [ "$1" == "" ]; then
 echo "..."
else
 echo "..."
fi
And don't mention RP in the log